core/txpool/blobpool/blobpool_test.go

```diff
@@ -49,8 +49,8 @@ import (
 
 var (
 	emptyBlob          = kzg4844.Blob{}
```

Comment:
Perhaps make this new(kzg4844.Blob), then the type will be correct and no need to ref it everywhere